manager/container.go

```diff
@@ -270,6 +273,44 @@ func (c *containerData) updateLoad(newLoad uint64) {
 	}
 }
 
+func (c *containerData) updateSubcontainerStats(stats *info.ContainerStats, path string) (*info.ContainerStats, error) {
```

Comment:
This list of subcontainers is a set of all descendants of the container, based on how it was populated in getSubcontainerContainerData in manager.go 708. But this would need to be recursive if that were to change to only immediate children. Why does the below need to only track immediate children rather than all descendant containers?